Quest stuff, food rebalance, config changes and buffed recipe

MikoTheBoi opened the pull request:
## What does this pull request do?

Adds a bunch of Quest QoL and quests, new recipes, rebalances food and changes some configs

## Changes made

- Added compatibility for recipes that use buckets to accept cells
- Added a new quest that explains droplets
- Added a new quest to explain overclockers
- Added more info to the Diving Suit (Yttr) quest
- Removed mention of insulated cables in the questbook
- Create now uses droplets with fractions in all displays
- Added a new questline for Astral Foods, Vinery, and Drink Beer
- Adjusted orders of some quests
- Changed the Threshold Switch to not use the Content Observer as a dependency
- Changed the Storage Unit quest to mention Tank Units as well
- Added new quests for the Generator, Chemical Reactor, Greenhouse, and Lightning Rod
- Buffed reward for Chapter 4 Rocket quest to give 4 Blaze Cakes and Thruster Boots
- Added the Sawmill as a chapter 4 quest (no recipes)
- Buffed the Iron Fence recipe by 2x
- Added new translations for Culinary Delights chapter
- Added new quest group for non main-progression quests
- Reverted Shimmer Apple recipe to use a Ender Crystal
- Changed Gamer's Delight recipe to Sequenced Assembly
- Nerfed Seared Potato recipe to use more Seared Stone and longer cool time 
- Blaze Fries now take heating
**WIP since Ethan's adding the skillet!**
- Fried Cod now takes heating
**WIP since Ethan's adding the skillet!**
- Buffed Astral Sauce hunger
- Buffed Quantum Pasta hunger
- Nerfed Dipped Potato by increasing no Jump time
- Buffed Compressed Onion hunger
- Buffed Food Amalgamation hunger 
- Nerfed Chocolate Ice Cream Speed time 
- Nerfed Gamer's Delight by increasing Shock time
- Buffed Blaze Fries And Cod Fire Resistance time
- Buffed Cod n' Blaze Fire Resistance and Absorption time
- Nerfed Integrated Circuit recipe so it's easier to read
**^^ To be reviewed might be an unnecessary nerf**

## Potential Compatibility Issues

New quest book group may conflict with other quest changes

MikoTheBoi committed:
Changed most bucket recipes to also accept their cell variants and vice versa
> Just a QoL fix for people using cells or buckets
Added rewards for completing the diving suit quest
> I thought it was just the type of quest to have it

MikoTheBoi committed:
Added new quest to explain droplets
> Many players got stuck on that. Moved over most info from the mc-general pin here

Added new quest to explain overclockers
> They're super useful so should be mentioned

Modified Power The World quest (Ch4 [Isn't it weird that chapters are ordered 1 higher than in game?] Quest28) to not mention insulated cables
> There are no insulated cables in Astral :clueless:

Modified Diving Suit quest (Ch6 Raow Quest29) to include more info
> Had to check out the YTTR wiki because of the lack of info. Added more and link to the wiki

MikoTheBoi committed:
Added a new questline for Astral Foods and more (some of vinery and CFB, have to add drinkbeer too)
> Players didn't know about it so quests should help with that!

Rebalanced some Astral Foods stuff
> Just recipe changes. Some didn't make sense

Added a new questgroup for non main progression (side quests)!
> Was discussed at some point. Declutters the QB a bit

More changes may've been made but I forgot lol

MikoTheBoi committed:
Moved all of Drink Beer and Vinery to Culinary Delights
> It's a food chapter. Frees up 0.5 a bit too

MikoTheBoi committed:
Fairly big quest changes! (will probably break other PRs)!

Adjusted the order index in some questlines due to the new Side Quest group

Changed the Threshold Switch quest to not use the Contest Observer as a dependency
> No reason why it's like this anymore

Changed the Storage Unit quest to mention Tank Units too
> Since they're both units I decided to put them in the same quest

Added new Ch3 quests (Generator, Chemical Reactor, Greenhouse and Lightning Rod)
> Weren't mentioned before. Hopefully people know about them now!

Moved some Ch3 quests to accomodate the new quests

Buffed Ch4 Rocket quest reward (4 Blaze Cakes and Thruster Boots)
> Now matches more to Ch2 and 3s rewards. Thrusters are just a nice utility

Added new Ch4 quest (Sawmill)
> The recipes are still disabled btw

Buffed Iron Fence recipe (1x->2x)
> HOLY they're expensive

Added new translations for Culinary Delights!

Disabled Create's Simplify Fluid Unit config (enables fractions)
> BIG change imo. Players can finally see the real fluid amounts in assembly recipes!!

MikoTheBoi commented:
Seems like some quests have major issues :(

The-Shortman commented:
Please actually type out your changelogs instead of saying "I'm too lazy" because then I have to do it instead and it's just a waste of time.

MikoTheBoi commented:
Added more info to the food rebalances and recipe changes!

MikoTheBoi committed:
Adjusted seared potato recipe
> Was a bit too expensive tbh
Nerfed IC recipe to consume 50mB per spout
> Easier to read. A silver bucket is now 5 ICs instead of 6. Might be a controversial change.

MikoTheBoi commented:
I don't think the massive changes that happened (FOR SOME REASON) actually broke anything!

MikoTheBoi committed:
Merge branch 'Astral-Experimental' into Astral-Experimental

ethanicusss committed:
fix snbt formatting issue with chapter 2 and 4

ethanicusss committed:
added a bit of specificity to sgc quest

ethanicusss committed:
TR upgrade extra info

ethanicusss committed:
Greenhouse controller info

ethanicusss commented:
I'm going to bed but quick question - any reason why some food items with hunger over 20 are being buffed? They already give over the max amount of hunger, so I assume this would have no effect

MikoTheBoi commented:
It's based on their recipe. For example the compressed onion should grant 243 hunger (based on the amount of onions) (I thought that was too much so 81 is close enough) and quantum pasta uses an astral singularity so it should be better than it

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2113,6 +2118,15 @@
   "ftbquests.chapter.chapter_3.quests80.description2": "Simply throw a valid item into Shimmer fluid and watch the SCIENCE unfold! Remember your protective goggles!",
+  "ftbquests.chapter.chapter_3.quests81.title": "What's a Droplet?",
+  "ftbquests.chapter.chapter_3.quests81.subtitle": "Whip out your calculator!",
+  "ftbquests.chapter.chapter_3.quests81.description0": "&6Droplets&r are a unit of fluid used by &6Hephaestus&r instead of &6mB&r!",
```
```suggestion
  "ftbquests.chapter.chapter_3.quests81.description0": "&6Droplets&r are a unit of fluid used by &6Fabric mods like Hephaestus&r, instead of &6mB&r!",
```
Would rather say "Fabric" instead of "Hephaestus" here as it is technically more accurate as all mods in Astral use droplets before converting to mB in REI. Maybe say "Fabric mods, like Hephaestus" or something similar.

MikoTheBoi replied:
Think the old version is easier for players to understand since REI wise, Hephaestus is the only mod to display in droplets (to my knowledge).

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests1.description0": "These are dishes coming from planet Earth (my home!).",
+  "ftbquests.chapter.culinary_delights.quests1.description1": "They're some of my most renowned dishes!",
+  "ftbquests.chapter.culinary_delights.quests2.title": "&bCooking 02:&r Up Above.",
+  "ftbquests.chapter.culinary_delights.quests2.description0": "With new locations, comes new cuisine! The moon is no exception to this!",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests2.description0": "With new locations, comes new cuisine! The Moon is no exception to this!",
```
A bit nitpicky but Moon should be capitalised

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests5.description1": "It also adds even more &7Resistance&r!",
+  "ftbquests.chapter.culinary_delights.quests5.description2": "Though the gold is &lreally&r heavy on the stomach.",
+  "ftbquests.chapter.culinary_delights.quests6.description0": "This is one of my beloved family recipes!",
+  "ftbquests.chapter.culinary_delights.quests6.description1": "The Potatos should get more &7Resistant&r after bathing for some time",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests6.description1": "The Potatoes should get more &7Resistant&r after bathing for some time",
```
Typo in "potatoes"

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests8.description0": "May conflict with Chocolate recipes. Use a List Filter to avoid that!",
+  "ftbquests.chapter.culinary_delights.quests9.description0": "The Ultimate &bS&r&eu&r&bg&r&ea&r&br&r &eR&r&bu&r&es&r&bh&r.",
+  "ftbquests.chapter.culinary_delights.quests10.description0": "I always get a case of nostalgia after these.",
+  "ftbquests.chapter.culinary_delights.quests10.description1": "As a kid I used &bZoom&r around the village after one!",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests10.description1": "As a kid I used to &bZoom&r around the village after one!",
```
I think there's a missing word here - "I used to zoom around the village..."

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests10.description0": "I always get a case of nostalgia after these.",
+  "ftbquests.chapter.culinary_delights.quests10.description1": "As a kid I used &bZoom&r around the village after one!",
+  "ftbquests.chapter.culinary_delights.quests11.subtitle": "Terraria? Haven't heard of it!",
+  "ftbquests.chapter.culinary_delights.quests11.description0": "Made this on accident. A happy accident!",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests11.description0": "Made this by accident. A happy accident!",
```
"By accident" is technically more correct than "on accident"

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests14.description2": "Maybe if one could pack them into a single item it would save a lot of time!",
+  "ftbquests.chapter.culinary_delights.quests14.description3": "Though personally I wouldn't eat that.",
+  "ftbquests.chapter.culinary_delights.quests15.description0": "Sure, regular Apples are alright but they need an extra hit to them!",
+  "ftbquests.chapter.culinary_delights.quests15.description1": "Shimmer's arlight but I need more!",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests15.description1": "Shimmer's alright but I need more!",
```
Typo in "alright"

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2982,6 +3022,62 @@
+  "ftbquests.chapter.culinary_delights.quests16.description2": "And someone mentioned feeling &cStronger&r afterwards?",
+  "ftbquests.chapter.culinary_delights.quests16.description3": "Worth noting down I suppose!",
+  "ftbquests.chapter.culinary_delights.quests17.description0": "Lost your space suit? This'll help you stay on the moon without a space suit!",
+  "ftbquests.chapter.culinary_delights.quests17.description1": "The company didn't hand out space suits for the whole crew so they made it the chef's (me) mission!",
```
```suggestion
  "ftbquests.chapter.culinary_delights.quests17.description1": "The company didn't hand out space suits for the whole crew so they made it the chef's (my) mission!",
```
The possessive form of first person pronoun makes more sense here in my opinion

The-Shortman commented on kubejs/assets/createastral/lang/en_us.json:
```diff
@@ -2614,6 +2628,27 @@
+  "ftbquests.chapter.chapter_4.quests140.tasks3.title": "Liquid XP Nuggies",
+  "ftbquests.chapter.chapter_4.quests141.subtitle": "Machines too slow? Use these!",
+  "ftbquests.chapter.chapter_4.quests141.description0": "&6Overclockers&r are upgrades for &6Tech Reborn&r machines.",
+  "ftbquests.chapter.chapter_4.quests141.description2": "They increase the speed of machines at the cost of making them use more energy.",
```
```suggestion
  "ftbquests.chapter.chapter_4.quests141.description2": "They increase the speed of machines at the cost of making them use more power.",
```
Power is technically more correct as overclockers increase energy consumption per unit time

MikoTheBoi committed:
Apply suggestions from code review

Added Short's suggestions

Co-authored-by: _Shortman <78680166+The-Shortman@users.noreply.github.com>
